### PR TITLE
Navbar added

### LINKED ISSUE #2925: Navbar with beautiful background
**Describe the project you want to add with tech stack**

I want to  make a new navbar.
It will be attractive.
Different colors will be used to make it more eye catching.
**Expected behavior**

**Screenshots (optional)**

It will look like this
image
Can you assign this task  to me?
**Additional context (optional )**

### PR DESCRIPTION
**Pull request by Shreyans2004**
# Title and Issue number 

closes
Close #2925

I have added the navbar .Can it be merged now?
# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by Shreyans2004** [comment]
@MohitGupta121 Can it be merged now?

File: Navbars/navbar_with_beautiful_background/image3.jpg
Comment (MohitGupta121) [inline Navbars/navbar_with_beautiful_background/image3.jpg:1]: @Shreyans2004 please do not upload this image in your PR, if your project is using an image upload it somewhere in the cloud and use that image link in your project. Thanks!

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by Shreyans2004** [comment]
@MohitGupta121  I have done the necessary changes. Can it be merged now?

File: Navbars/navbar_with_beautiful_background/image3.jpg
Comment (Shreyans2004) [reply Navbars/navbar_with_beautiful_background/image3.jpg:1]: @MohitGupta121 I have done the necessary changes. Can it be merged now?

**Review by MohitGupta121 — APPROVED** [review]

**Pull request merged**